lib/stores/AuthStore.ts

```diff
@@ -40,17 +42,25 @@ export class AuthStore {
     this.oAuthType = oAuthType
     this.clientId = clientIdEnum[this.oAuthType]
 
-    // Get AuthToken
-    const response = await getAuthToken(client)({
-      clientId: this.clientId,
-      oauthType: this.oAuthType,
-      code,
-      redirectUri: redirect_url
-    })
-    const accessToken = response.data.oAuthTokenAuth.token
-    this.setAccessToken(accessToken)
+    const localStorageCode = localStorage.getItem(CODE)
+    const localStorageAccessCode = localStorage.getItem(TOKEN_KEY)
+
+    if (!localStorageCode && !localStorageAccessCode) {
```

Comment:
기존 코드랑 다를 때만 호출하게 하는 것은 어떨까요?
한번 호출한 code는 invalid하다는 것이 더 드러날 수 있을 것 같아요 :smile:

```suggestion
    if (localStorageCode!==code && !localStorageAccessCode) {
```